lib/conversation/replies.js

```diff
@@ -95,7 +104,7 @@ function renderContinueConversationMsg(fn) {
         // Store current campaign to continue conversation in subsequent messages.
         // TODO: Should this be handled more eloquently?
         args.req.user.current_campaign = args.req.campaignId;
```

Comment:
I was debating whether it made sense to pass the `req.campaignId` as a second optional parameter to `setLastOutboundTemplate` to save it there... and whether it would add/hurt eloquence...